### ISSUE TITLE
[typescript template] not working - help wanted

### ISSUE THREAD

**Issue by FranciscoKloganB** [viandwi24/nuxt3-awesome-starter#53]
Just downloaded the project. Installed all dependencies with `pnpm`, shamefully hoist and all those caveats known in PNPM.

Changed a few things in About page and then I noticed something.

I am getting Type checking in `ts` files. I am also getting Type checking within `.vue` files within the `<script>` tags.

![image](https://user-images.githubusercontent.com/25959244/222256849-17f8c655-97ae-4286-913a-b1159414a1b3.png)

However, when it comes to getting Type checking I do not get any type checking regarding props of imported components. I only get type inference for things within the "self" script tag.

See below:

![image](https://user-images.githubusercontent.com/25959244/222257059-284fcea7-7b71-43c3-bc66-9502b49b88d5.png)
![image](https://user-images.githubusercontent.com/25959244/222257095-c23dbd12-07d5-41d7-a7c8-64f3b852f4d9.png)

Basically, any component that is external to the current file, used in template is being typed as `any`. I have no clue why this is happening. Do you have any ideia?

**Comment by FranciscoKloganB** [comment]
Uninstalled `@nuxt/content`, the problem is fixed.

I also appears is a known issue, but no answers from codeowners: https://github.com/nuxt/content/issues/1358
